packages/os/prairiedog-toml

```diff
@@ -0,0 +1,23 @@
+[required-extensions]
+std = { version = "v1", helpers = ["default"]}
```

Comment:
Looks like we don't actually use `default`. I think you can remove the dependency on `std`.